src/web/app/components/question-types/question-statistics/question-statistics-calculation/constsum-recipients-question-statistics-calculation.ts

```diff
@@ -19,6 +19,9 @@ export class ConstsumRecipientsQuestionStatisticsCalculation
   pointsPerOption: Record<string, number[]> = {};
   totalPointsPerOption: Record<string, number> = {};
   averagePointsPerOption: Record<string, number> = {};
+  averagePointsExcludingSelf: Record<string, number> = {};
+  totalSum: number = 0;
+  totalStudents: number = 0;
```

Comment:
(On a sidenote, it might not be a good idea to have variables like these used temporarily for computation only declared as class properties! Variables should try to live in the smallest scope possible.)

Reply:
I see! Thank you for the guidance!